shop-service/product-service/mockData/index.js

```diff
@@ -0,0 +1,28 @@
+const PRODUCTS_LIST = [
```

Comment:
only 3 items here. Where are the rest of data?

Reply:
done